src/test/scala/mesosphere/marathon/integration/LeadershipAbdicationIntegrationTest.scala

```diff
@@ -0,0 +1,90 @@
+package mesosphere.marathon
+package integration
+
+import mesosphere.marathon.Protos.Constraint.Operator
+import mesosphere.marathon.integration.setup.WaitTestSupport
+
+import scala.concurrent.duration._
+
+/**
+  * Regression Test for https://github.com/mesosphere/marathon/issues/3783
+  *
+  * Adapted from https://github.com/EvanKrall/reproduce_marathon_issue_3783
+  */
+@IntegrationTest
+class LeadershipAbdicationIntegrationTest extends LeaderIntegrationTest {
+
+  after(cleanUp())
+
+  override val numAdditionalMarathons = 5
```

Comment:
A) this number is pretty high
B) the number of abdications is the number of additional marathons - right?

Can we write this test in a way, that this number controls both.
Perhaps we can lower this number?